__DEFINES/process.dm

```diff
@@ -0,0 +1,15 @@
+// Process status defines
+#define PROCESS_STATUS_IDLE 1
+#define PROCESS_STATUS_QUEUED 2
+#define PROCESS_STATUS_RUNNING 3
+#define PROCESS_STATUS_MAYBE_HUNG 4
+#define PROCESS_STATUS_PROBABLY_HUNG 5
+#define PROCESS_STATUS_HUNG 6
+
+// Process time thresholds
+#define PROCESS_DEFAULT_HANG_WARNING_TIME 300 // 30 seconds
```

Comment:
use `30 SECONDS` and such where applicable

Reply:
Fixed in #30178

Reply:
File since removed, was unused